changelog/10241.bugfix.md

```diff
@@ -0,0 +1 @@
+[#10241](https://github.com/RasaHQ/rasa/issues/10241): Bump TensorFlow version to 2.6.2.
```

Comment:
💡 changelog fragments should not include links to issues. Otherwise it leads to issues like [this one](https://github.com/RasaHQ/rasa/issues/10452). The link to the issue is automatically added from the the name of the changlog fragment file. Please refer to [the documentation here](https://github.com/RasaHQ/rasa/blob/main/changelog/README.md), when in doubt 😄

Reply:
@m-vdb I'm sorry for the mistake. Is it too late to change it?

Reply:
it's OK I'm fixing it

Reply:
thanks!